python/pipeline/stack.py

```diff
@@ -21,8 +21,9 @@
 downwards, y in the second axis pointing towards you and x on the third axis pointing to 
 the right.
 """
-dj.config['external-stack'] = {'protocol': 'file',
-                               'location': '/mnt/dj-stor01/pipeline-externals'}
+dj.config['stores'] = {'external-stack': {'protocol': 'file', 'location': '/mnt/dj-stor01/pipeline-externals'}}
```

Comment:
I don't think it's necessary to call the storage `external-` something. Rather something like `stack_storage`. Also, would be best for the `location` to be configurable via environment variable, so it's not hard coded into the code.

Reply:
thanks, fixed the storage naming.

good idea, is it best practice to declare the environment variable in `docker-compose.yml`?